File: uri/C/beginner/1134.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){

	int contador, gasolina = 0, alcool = 0, diesel = 0;

	do{
		cin >> contador;
		if(contador == 1){
			alcool++;
		}
		else if(contador == 2){
			gasolina++;
		}
		else if(contador == 3){
			diesel++;
		}
		else if(contador == 4){
			break;
		}
	}while(contador != 1 || contador != 2 || contador != 3);

	printf("MUITO OBRIGADO\n");
	printf("Alcool: %d\n", alcool);
	printf("Gasolina: %d\n", gasolina);
	printf("Diesel: %d\n", diesel);

	return 0;
}

File: uri/python/1134.py
# -*- coding: utf-8 -*-

combustivel = 1
alcool = 0
gasolina = 0
diesel = 0

while combustivel != 4:
    combustivel = 0
    combustivel = int(input())
    if combustivel == 1 or combustivel == 2 or combustivel == 3 or combustivel == 4:
        if combustivel == 1:
            alcool += 1
        elif combustivel == 2:
            gasolina += 1
        elif combustivel == 3:
            diesel += 1
 
print('MUITO OBRIGADO')
print('Alcool: {}'.format(alcool))
print('Gasolina: {}'.format(gasolina))
print('Diesel: {}'.format(diesel))

